src/index.js

```diff
@@ -18,30 +18,44 @@ domReady(() => {
 });
 
 const pages = [
-  { key: "/store", Page: Store },
-  { key: "/store/products", Page: Products },
-  { key: "/store/details", Page: StoreDetails },
-  { key: "/store/sales_discounts", Page: WonderCart },
-  { key: "/store/payments", Page: AllPayments },
+  { key: '/store', Page: Store },
+  { key: '/store/products', Page: Products },
+  { key: '/store/details', Page: StoreDetails },
+  { key: '/store/sales_discounts', Page: WonderCart },
+  { key: '/store/payments', Page: AllPayments },
 ];
 
 function parseWCStatus(data) {
   const status = data?.details?.woocommerce?.status;
-  const isActive = status === "active";
-  const needsInstall = status === "need_to_install";
-  const isInstalling = data?.queue?.includes("woocommerce");
+  const isActive = status === 'active';
+  const needsInstall = status === 'need_to_install';
+  const isInstalling = data?.queue?.includes('woocommerce');
   return { isActive, needsInstall, isInstalling };
 }
 
 /** @type {import("..").NewfoldEcommerce}  */
 export function NewfoldECommerce(props) {
   let { data: woo, mutate } = useSWR(
-    "woo-status",
-    () => PluginsSdk.queries.status("woocommerce").then(parseWCStatus),
+    'woo-status',
+    () => PluginsSdk.queries.status('woocommerce').then(parseWCStatus),
     { revalidateOnReconnect: false, refreshInterval: 30 * 1000 }
   );
   let { Page } =
     pages.find((page) => page.key === props.state.location) ?? pages[0];
+  let { notify } = props.wpModules;
+
+  useEffect(() => {
+    sessionStorage.getItem('reload') === 'true' &&
+      notify.push('woo-install-status', {
+        title: __(
```

Comment:
@manikantakailasa Please import `@wordpress/i18n` else translations for push notifications won't work.